internal/schema2/virtual_pci_device.go

```diff
@@ -0,0 +1,16 @@
+/*
+ * HCS API
+ *
+ * No description provided (generated by Swagger Codegen https://github.com/swagger-api/swagger-codegen)
+ *
+ * API version: 2.3
+ * Generated by: Swagger Codegen (https://github.com/swagger-api/swagger-codegen.git)
+ */
+
+package hcsschema
+
+// TODO: This is pre-release support in schema 2.3. Need to add build number
+// docs when a public build with this is out.
+type VirtualPciDevice struct {
```

Comment:
`PCI`

Reply:
This is generated from hcs so I didn't mess with it. Is it okay to change the names?

Reply:
Leave the generated names unless you want to stop using the generated types. Otherwise this will be a merge nightmare